{{cookiecutter.repo_name}}/LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright {{cookiecutter.year}} {{cookiecutter.author_full_name}}
```

Comment:
This is just instructions for how to apply the license, e.g. in your readme file. The `LICENSE` file itself should not be changed.

Reply:
ok I understand, thank.